.babelrc

```diff
@@ -1,11 +1,16 @@
 {
-	"presets": ["@wordpress/babel-preset-default"],
+	"presets": [
+		"@wordpress/babel-preset-default"
+	],
 	"plugins": [
 		[
 			"@wordpress/babel-plugin-makepot",
 			{
 				"output": "languages/coblocks-js.pot"
 			}
 		]
+	],
+	"test": [
```

Comment:
I don't think this is supposed to be here. Babel is throwing an error in the build process and indicating `"test"` is not a valid key for `.babelrc`.